src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let numcall = 0;
```

Comment:
let's give our vars more descriptive names
```suggestion
  let callsLimit = 0;
```